notebooks/Practice.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
How is this notebook supposed to be used? Based on your source here, it is not being used in any way. Would you expect to include it in the image or are you using this just for debugging? I'd try to think about how you are intending on using this, update the `Dockerfile`/`docker-compose.yaml` appropriately based on that, and document it properly in your `README.md` so others can easily follow.

Reply:
As of right now, the notebook is being used for development. We will mount this in a volume in the `docker-compose` if we want to access this later on.